binary_search_tree/binary_search_tree.py

```diff
@@ -5,13 +5,59 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    # check if the new node's value is less than our current node's value
+    if value < self.value:
+      # if there's no left child here already, place the new node here
+      if not self.left:
+        self.left = BinarySearchTree(value)
+      else:
+        # otherwise, repeat the process!
+        self.left.insert(value)
+    # check if the new node's value is greater than or equal to our current node's value
+    elif value >= self.value:
+      # if there's no right child here already, place the new node here
+      if not self.right:
+        self.right = BinarySearchTree(value)
+      else:
+        # otherwise, repeat the process!
+        self.right.insert(value)
 
   def contains(self, target):
-    pass
+    if target == self.value:
+       return True
+
+    if target < self.value :
+      if not self.left:
+        pass # do nothing because it is recursive loop
+      else:
+        if self.left.contains(target):
```

Comment:
not sure what you are trying to check here.
you should return that code rather than having it as a if